server/hotrod/src/main/scala/org/infinispan/server/hotrod/HotRodServer.scala

```diff
@@ -247,7 +248,19 @@ class HotRodServer extends AbstractProtocolServer("HotRod") with Log {
       clientListenerRegistry.setEventMarshaller(Option(marshaller))
    }
 
-   override def stop: Unit = {
+   def addCacheStore(name: String, instance: Object): Unit = {
```

Comment:
Regarding to turning on and off persistence on running system - I think this is a great idea and it makes a good fit into RHQ management console use case. However I would prefer to create a separate JIRA for that, since it may result in major refactoring.